packages/paima-sdk/crypto/src/chains/algorand.ts

```diff
@@ -6,6 +6,34 @@ import {
 } from "@paima/utils";
 import type { IVerify } from "../IVerify.ts";
 import { Value } from "@sinclair/typebox/value";
+import { utf8ToHex } from "web3-utils";
+import {
+  type Transaction,
+  type SuggestedParams,
+  makePaymentTxnWithSuggestedParamsFromObject,
+  decodeSignedTransaction,
+  type SignedTransaction,
+} from "algosdk";
```

Comment:
as a reminder, we were trying to avoid static imports for these because 99% of dApps will not use Algorand at all, and it saves a lot of app startup time to not include different chains (both because of reduction in bundled file size, but also because some of these libraries initialize a bunch of WASM stuff / crypto stuff on load)